src/scripts/Player.ts

```diff
@@ -71,7 +71,8 @@ class Player {
         // TODO(@Isha) move to underground classes.
         const mineInventory = (savedPlayer.mineInventory || [])
             // TODO: Convert this to object spread after we're on TS modules
-            .map((v) => Object.assign({}, v, { amount: ko.observable(v.amount) }));
+            .map((v) => Object.assign({}, v, { amount: ko.observable(v.amount) }))
+            .map((v) => Object.assign({}, v, { sellLocked: ko.observable(v.sellLocked) }));
```

Comment:
I haven't tested this, but I think it should work without mapping twice?
```suggestion
            .map((v) => Object.assign({}, v, {
                amount: ko.observable(v.amount),
                sellLocked: ko.observable(v.sellLocked),
            }));
```